include/camp/resource.hpp

```diff
@@ -226,6 +239,11 @@ namespace resources
       Res resource_;
     };
 
+    bool operator== (camp::resources::Resource& l, camp::resources::Resource& r)
```

Comment:
Maybe `==` for strongly typed as well, so a strongly typed instance doesn't have to be wrapped to be compared?

Reply:
I think Mike's changes addressed this as there is no longer a floating operator overload there, it's been pulled into the class instead... thoughts?